files/plir-hapifhir/deploy.env

```diff
@@ -0,0 +1 @@
+DESTROY_VOLUMES=false
```

Comment:
So you are marking you want backups, is that correct?

Reply:
sure